language-support/hs/bindings/src/DA/Ledger/Jwt.hs

```diff
@@ -0,0 +1,23 @@
+-- Copyright (c) 2019 The DAML Authors. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module DA.Ledger.Jwt(
+  Jwt,
+  tryCreateFromString,
+  toString
+  ) where
+
+import Data.List.Extra (splitOn)
```

Comment:
Why do we depend on the `jwt` library but then have our own type and validation logic instead of using something like jwt’s [decode](https://hackage.haskell.org/package/jwt-0.10.0/docs/Web-JWT.html#v:decode)?